extensions/amp-social-share/1.0/social-share.js

```diff
@@ -160,11 +160,11 @@ function handleActivation(finalEndpoint, target) {
         `Could not complete system share.  Navigator unavailable. ${NAME}`
       );
     }
-  } else if (protocol === 'sms') {
+  } else if (protocol === 'sms' || protocol === 'mailto') {
     openWindowDialog(
       window,
-      finalEndpoint.replace('?', '?&'),
-      target,
+      protocol === 'sms' ? finalEndpoint.replace('?', '?&') : finalEndpoint,
+      isIos() ? '_top' : target,
```

Comment:
I also expected this logic to be symmetrically removed from AMP's side. No?

Reply:
Makes sense.  I was wondering though, is there a possible situation where only the AMP environment has access to `window.navigator.userAgent` (and Preact side does not)?  If so, I was thinking the AMP side could send the right target, but maybe this is not a valid use-case.